### PR TITLE
Simplified the details around processor documentation to be more clear

### LINKED ISSUE #199: Processor with FOSUserBundle
Could I have information on what the [example of processor](https://github.com/nelmio/alice#processors) given really does? Is it really necessary because I forgot to use it and it seemed to work so just asking to clear things up.

### PR DESCRIPTION
**Pull request by tshelburne**
fixes #199 
Also references #200

### CODE REVIEW

**Comment by tshelburne** [comment]
@theofidry Ping - thoughts?

File: src/Nelmio/Alice/Instances/Processor/Methods/Current.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+     */
+    public function canProcess(ProcessableInterface $processable)
+    {
+        return is_string($processable->getValue());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function process(ProcessableInterface $processable, array $variables)
```
Comment (theofidry) [inline src/Nelmio/Alice/Instances/Processor/Methods/Current.php]: Fix indentation?

**Comment by theofidry** [comment]
Aside from nitpicking (see the note), it looks good to me :)

File: src/Nelmio/Alice/Instances/Processor/Methods/Current.php (hunk 1 above)
Comment (tshelburne) [inline src/Nelmio/Alice/Instances/Processor/Methods/Current.php]: Wow, this must have been a legacy file from way back - not sure, will remove.

**Comment by tshelburne** [comment]
@theofidry Well that was embarassing - always do spike work in a branch.

**Comment by theofidry** [comment]
It's a rough day^^

**Pull request merged**